### PR TITLE
Zitong Yang leader board submission

### PR DESCRIPTION
**Pull request by ZitongYang**
* Final validation loss 3.91.
* `wandb` link [https://api.wandb.ai/links/zitongyang/j0fzs8tn](url)
* I started by trying MoE and tying embedding and lm head, but didn't get good result. Eventually, I tuned the hyperparameter toward larger batch size and shorter context length to maximize throughput and that gives best performance.

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
